labelbox/client.py

```diff
@@ -1597,3 +1597,49 @@ def get_model_slice(self, slice_id) -> ModelSlice:
         """
         res = self.execute(query_str, {"id": slice_id})
         return Entity.ModelSlice(self, res["getSavedQuery"])
+
+    class RemoveFeatureFromOntologyResult:
+        archived: bool
+        deleted: bool
+
+    def remove_feature_schema_from_ontology(
```

Comment:
Any reason for using `remove` over `delete` verb? This goes for both the function name, return type and wherever else the term `remove` was used over `delete`, like in the logger messages

Reply:
No strong reason other than I felt that since a node can sometimes be archived as opposed to deleted, "removed" felt like a more generic term that describes the behavior, but I don't have a strong opinion. Happy to change it `delete` for consistency.

Reply:
[Updated](https://github.com/Labelbox/labelbox-python/pull/958/commits/50fa1cbe47d9fc63e9c7587c4f148da5c9487c89)